eng/scripts/Split-Tests-Xml.ps1

```diff
@@ -0,0 +1,66 @@
+<#
+.SYNOPSIS
+Split the test dependencies into different project files.
+
+.DESCRIPTION
+.NET pipelines will run centain number of test dependencies in each entry of job matrix. 
+The Script is to split all test dependencies into difference sets for later use. 
+
+.PARAMETER ProjectFilePath
+The file path contains all test dependencies.
+
+.PARAMETER OutputFolder
+The outputFolder which store all the files after splitting.
+
+.PARAMETER NumOfTestsPerJob
+The number of test run on each job.
+
+.EXAMPLE
+Run script with default parameters.
+
+Split-Tests-Xml.ps1 -OutputFolder "projects" -ProjectFilePath "eng/projects.txt"
+
+#>
+Param (
+    [string] $ProjectFilePath,
+    [string] $OutputFolder  = "projects",
+    [int] $NumOfTestsPerJob = 20
+)
+
+$packages = Get-Content $ProjectFilePath
+$numOfGroups = [Math]::Floor([decimal]($packages.Count / $NumOfTestsPerJob ))
+$pkgGroup = [System.Collections.ArrayList]::new()
+$i = 0
+for ($i = 0; $i -lt $numOfGroups; $i++) { 
+  $pkgGroup.Add($packages[( $i * $NumOfTestsPerJob )..(($i + 1) * $NumOfTestsPerJob )])
+}
+if ($packages.Count -gt $numOfGroups * $NumOfTestsPerJob) {
+  $pkgGroup.Add($packages[($numOfGroups * $NumOfTestsPerJob)..$packages.Count])
+}
+
+$null = New-Item -Path $OutputFolder -ItemType "directory" -Force 
+# Retain the structure without ProjectReference.
+$templateXml=[xml]'<Project></Project>'  
+$projectNode = $templateXml.SelectNodes("/Project")
+$itemGroupNode=$templateXml.CreateNode("element", "ItemGroup", "")
+$projectNode.AppendChild($itemGroupNode)
+
+# Store the itemGroup in template file
+$templateFile = "eng/template.xml"
```

Comment:
Do we really need to save a template file? We can just add the wrapper xml steps in the loop below.